backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -67,7 +67,9 @@ public async Task<ActionResult<LexAuthUser>> RegisterAccount(RegisterAccountInpu
             return ValidationProblem(ModelState);
         }
 
-        var userEntity = CreateUserEntity(accountInput, emailVerified: false);
+        var jwtUser = _loggedInContext.MaybeUser;
+
+        var userEntity = CreateUserEntity(accountInput, emailVerified: false, jwtUser ?? null);
```

Comment:
`jwtUser ?? null` is not needed. `??` means that it'll fallback to the other value if the left side is null, but we want null in this case.